tests/test_wagon.py

```diff
@@ -862,12 +794,6 @@ def test_fail_validation_exclude_and_missing_wheel(self):
             if os.path.isfile(archive_name):
                 os.remove(archive_name)
 
-    @mock.patch('wagon.IS_VIRTUALENV_INSTALLED', False)
-    def test_create_with_validation_no_virtualenv_installed(self):
```

Comment:
this is the same test that checks the "assert_virtualenv" function, while mocking out the only thing that matters for it